v2/cmd/asoctl/cmd/import_azure_resource.go

```diff
@@ -47,6 +47,25 @@ func newImportAzureResourceCommand() *cobra.Command {
 
 	cmd.MarkFlagsMutuallyExclusive("output", "output-folder")
 
+	cmd.Flags().StringVarP(
+		&options.namespace,
+		"namespace",
+		"n",
+		"",
+		"Write the imported resources to the specified namespace")
+	cmd.Flags().StringSliceVarP(
+		&options.labels,
+		"label",
```

Comment:
I'm torn between `label` and `labels` here -- same for annotations.

Thoughts?

Reply:
Definitely singular. 

The only plural CLI commands I can think of are ones that only ever deal with many things, and these options do need to support singles.